go.mod

```diff
@@ -173,3 +174,5 @@ require (
 )
 
 replace github.com/foogod/go-powerwall => github.com/andig/go-powerwall v0.2.1-0.20220205120646-e5220ad9a9a0
+
+replace github.com/evcc-io/eebus => github.com/DerAndereAndi/eebus v0.0.0-20220928183324-ef68aab01342
```

Comment:
Können wir den vorab mergen?

Reply:
Das sollte gemacht werden. Deshalb ist der PR dort nicht mehr im Draft modus aber dieser hier schon

Reply:
Kann ich den EEBus PR schon mergen ohne dass der evcc Build kaputt geht? Dann wär das der nächste Schritt?

Reply:
Klar, es wird ja in master ein älterer Commit referenziert

Reply:
Stimm auch wieder. Danke :)